src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -1125,13 +1145,24 @@ protected TableTimer createTableTimer(String name, Timer keyspaceTimer)
     }
 
     protected TableTimer createTableTimer(String name, String alias, Timer keyspaceTimer)
+    {
+        return createTableTimer(name, alias, null, keyspaceTimer);
+    }
+
+    protected TableTimer createTableTimer(String name, String alias, String deprecated, Timer keyspaceTimer)
```

Comment:
Another case here where I don't think `name` and `alias` are ever not identical, and the `createTableTimer(String name, String alias, Timer keyspaceTimer)` overload essentially should just go away.